lib/date_time/date_time.c

```diff
@@ -631,19 +283,23 @@ int date_time_uptime_to_unix_time_ms(int64_t *uptime)
 
 	uptime_prev = *uptime;
 
-	if (!initial_valid_time) {
+	if (!date_time_is_valid()) {
```

Comment:
This check is not needed since it is included in `date_time_now`

Reply:
What about when you call `date_time_uptime_to_unix_time_ms` directly without `date_time_now`?

Reply:
I'm referring to the added `date_time_now()` call on line 291. It has the exact same check using `date_time_is_valid` at the beginning of the function. So essentially you have two identical checks in a row when calling `date_time_uptime_to_unix_time_ms`. Am I missing something?

Reply:
True, removing this check here.